marlowe-runtime-web/src/Language/Marlowe/Runtime/Web/Types.hs

```diff
@@ -308,14 +331,16 @@ data ContractState = ContractState
   , currentContract :: Maybe Semantics.Contract
   , state :: Maybe Semantics.State
   , utxo :: Maybe TxOutRef
+  , assets :: Assets
   , txBody :: Maybe TextEnvelope
   , unclaimedPayouts :: [Payout]
   }
   deriving (Show, Eq, Generic)
 
 data Payout = Payout
   { payoutId :: TxOutRef
-  , role :: Text -- TODO (N.H) : add assets that will be retrieved by the payout
+  , role :: Text
```

Comment:
did we say we would like to have an `AssetId` here ?

Reply:
You are right; I will make that change when I implement https://input-output.atlassian.net/browse/PLT-7204.